demo/jvm-batik/src/main/kotlin/frontendContextDemo/scripts/LimitsContinuous.kt

```diff
@@ -47,6 +45,22 @@ object LimitsContinuous {
                     geom_vline(linetype = 3) { xintercept = Pair(-10, 10) } +
                     geom_hline(linetype = 3) { yintercept = listOf(-10, 10) }
                     ).show()
+
+            // scale continuous limits
+
+            (p + scale_x_continuous(limits = -1 to 1) +
+                    geom_vline(linetype = 3) { xintercept = -1 to 1 }
+                    ).show()
+
+            (p + scale_x_continuous(limits = -1 to 1, expand = listOf(.0, .5)) +
+                    geom_vline(linetype = 3) { xintercept = -1 to 1 }
+                    ).show()
+
+            (p + scale_x_continuous(limits = -1 to 1, expand = listOf(.0, .5)) +
+                    geom_vline(linetype = 3) { xintercept = -1 to 1 } +
+                    scale_y_continuous(limits = -2 to 2, expand = listOf(.0, 1.0)) +
```

Comment:
Pls try if `expand = listOf(.0, 1)` works.

Reply:
It works, example is updated.